include/swift/Basic/LangOptions.h

```diff
@@ -147,6 +147,11 @@ namespace swift {
     /// configuration options.
     bool EnableObjCInterop = true;
 
+    /// Enable C++ interop code generation and build configuration
+    /// options.
+    /// FIXME: Disabled by default until this is fully baked.
```

Comment:
I think it may be even longer than that, since C++ *does* change the view of existing headers, meaning that turning it on may be a source-breaking change.